core/arch/arm/kernel/spmc_sp_handler.c

```diff
@@ -52,6 +57,177 @@ static TEE_Result ffa_get_dst(struct thread_smc_args *args,
 	return FFA_OK;
 }
 
+static int add_mem_region_to_sp(struct ffa_mem_access *mem_acc,
+				struct sp_mem *smem)
```

Comment:
These unused parameters are bound to cause compiler warnings, please fix here and below.
Note that all commits are supposed to compile and not cause regressions in order to make it easier to bisect.

Reply:
Done